VARIABLES.md

```diff
@@ -1742,7 +1790,7 @@ Below are the supported variables for the role confluent.kafka_connect
 
 ### kafka_connect_custom_log4j
 
-Boolean to enable cp-ansible's Custom Log4j Configuration
+Boolean to reconfigure Connect's logging with RollingFileAppender and log cleanup
```

Comment:
Boolean to reconfigure Kafka Connect's logging with the RollingFileAppender and log cleanup functionality.